sentry.conf.py

```diff
@@ -0,0 +1,309 @@
+# This file is just Python, with a touch of Django which means
+# you can inherit and tweak settings to your hearts content.
+
+# For Docker, the following environment variables are supported:
+#  SENTRY_POSTGRES_HOST
+#  SENTRY_POSTGRES_PORT
+#  SENTRY_DB_NAME
+#  SENTRY_DB_USER
+#  SENTRY_DB_PASSWORD
+#  SENTRY_RABBITMQ_HOST
+#  SENTRY_RABBITMQ_USERNAME
+#  SENTRY_RABBITMQ_PASSWORD
+#  SENTRY_RABBITMQ_VHOST
+#  SENTRY_REDIS_HOST
+#  SENTRY_REDIS_PASSWORD
+#  SENTRY_REDIS_PORT
+#  SENTRY_REDIS_DB
+#  SENTRY_MEMCACHED_HOST
+#  SENTRY_MEMCACHED_PORT
+#  SENTRY_FILESTORE_DIR
+#  SENTRY_SERVER_EMAIL
+#  SENTRY_EMAIL_HOST
+#  SENTRY_EMAIL_PORT
+#  SENTRY_EMAIL_USER
+#  SENTRY_EMAIL_PASSWORD
+#  SENTRY_EMAIL_USE_TLS
+#  SENTRY_ENABLE_EMAIL_REPLIES
+#  SENTRY_SMTP_HOSTNAME
+#  SENTRY_MAILGUN_API_KEY
+#  SENTRY_SINGLE_ORGANIZATION
+#  SENTRY_SECRET_KEY
+#  GITHUB_APP_ID
+#  GITHUB_API_SECRET
+#  BITBUCKET_CONSUMER_KEY
+#  BITBUCKET_CONSUMER_SECRET
+from sentry.conf.server import *  # NOQA
+
+import os
+import os.path
+
+CONF_ROOT = os.path.dirname(__file__)
+
+postgres = env('SENTRY_POSTGRES_HOST') or (env('POSTGRES_PORT_5432_TCP_ADDR') and 'postgres')
+if postgres:
+    DATABASES = {
+        'default': {
+            'ENGINE': 'sentry.db.postgres',
+            'NAME': (
+                env('SENTRY_DB_NAME')
+                or env('POSTGRES_ENV_POSTGRES_USER')
+                or 'postgres'
+            ),
+            'USER': (
+                env('SENTRY_DB_USER')
+                or env('POSTGRES_ENV_POSTGRES_USER')
+                or 'postgres'
+            ),
+            'PASSWORD': (
+                env('SENTRY_DB_PASSWORD')
+                or env('POSTGRES_ENV_POSTGRES_PASSWORD')
+                or ''
+            ),
+            'HOST': postgres,
+            'PORT': (
+                env('SENTRY_POSTGRES_PORT')
```

Comment:
why not use `env('SENTRY_POSTGRES_PORT', '')`?

Reply:
I used the default `sentry.conf.py` and added `THALIA_API_DOMAIN` and `THALIA_API_SECRET`. 

I changed the `env` calls (using one `or`) to use the default value parameter.

Reply:
weird that they have it like that. Might be something for a PR on Sentry.